atomate/utils/database.py

```diff
@@ -212,15 +240,20 @@ def _get_s3_store(self, store_name):
         Args:
             store_name: correspond to the the key within calcs_reversed.0 that will be stored
         """
-        index_store_ = MongoStore(
-            database=self.db_name,
-            collection_name=f"{self.maggma_store_prefix}_{store_name}_index",
-            host=self.host,
-            port=self.port,
-            username=self.user,
-            password=self.password,
-            key = 'fs_id'
-        )
+        if self.host_uri is not None:
+            index_store_ = MongoURIStore(uri=self.host_uri, database=self.db_name,
+                                         collection_name = f"{self.maggma_store_prefix}_{store_name}_index",
+                                         )
```

Comment:
Can you tidy this too pls.
```suggestion
            index_store_ = MongoURIStore(
                uri=self.host_uri,
                database=self.db_name,
                collection_name = f"{self.maggma_store_prefix}_{store_name}_index",
            )
```